src/test/java/com/parrit/controllers/PersonControllerTest.java

```diff
@@ -186,20 +186,6 @@ public void movePerson_movesMatchingPersonToGivenPairingBoardId_andReturnsTheUpd
         verify(mockProjectRepository).save(expectedProject);
     }
 
-    @Test
-    public void movePerson_whenPersonPositionIsFloatingIsNull_returnsError() throws Exception {
-        PersonPositionDTO personPositionDTO = new PersonPositionDTO();
-        personPositionDTO.setFloating(null);
-        personPositionDTO.setPairingBoardId(88L);
-
-        mockMvc.perform(put("/api/project/1/person/76/position")
-                .contentType(MediaType.APPLICATION_JSON)
-                .content(objectMapper.writeValueAsString(personPositionDTO)))
-                .andExpect(status().isBadRequest())
-                .andExpect(jsonPath("$.message", equalTo(null)))
-                .andExpect(jsonPath("$.fieldErrors.floating", equalTo("Where are you trying to go? Floating or a pairing board?")));
-    }
-
```

Comment:
The field floating is changed to boolean from Boolean so it can't be null any more.